ci/fats.sh

```diff
@@ -60,7 +60,13 @@ travis_fold end system-install
 # run test functions
 source $fats_dir/functions/helpers.sh
 
-for test in command; do
+if [ "$mode" = "full" ]; then
+  functions=(command)
```

Comment:
Should we run everything in full mode and a subset in lite mode? This seems inverted to me.

Reply:
yep, 🤦‍♂️